ibis/backends/pandas/udf.py

```diff
@@ -269,7 +269,7 @@ def aggregator(first, *rest):
                 # because this is the inner loop and we do not want
                 # to wrap a scalar value with a series.
                 if isinstance(op._output_type, dt.Struct):
-                    return coerce_to_dataframe(result, op._output_type.names)
```

Comment:
Can we remove this since coercion happens in the udf wrapper?

Reply:
Same for the pyspark wrapper

Reply:
Removed both